tools/Generator/Xamarin.Forms.Core.json

```diff
@@ -95,18 +95,6 @@
           "name": "Scale",
           "defaultValue": "1.0"
         },
-        {
-          "name": "Style",
```

Comment:
Is it supposed to be removed?

Reply:
Along with StyleClass

Reply:
I took it from fuget:
```fsharp
    public IList<string> class { get; set; }
    public Style Style { get; set; }
    public IList<string> StyleClass { get; set; }
```
In the Visualelement it looks like it does not exist anylonger

Reply:
It has been moved to NavigableElement now, which VisualElement inherits
https://docs.microsoft.com/en-us/dotnet/api/xamarin.forms.navigableelement?view=xamarin-forms

Reply:
Comparing 3.4.0 vs 4.0.0, fuget shows that NavigableElement was created with these 3 properties
https://www.fuget.org/packages/Xamarin.Forms/4.0.0.425677/lib/netstandard2.0/diff/3.4.0.1008975/

Reply:
Maybe we will need to change the Generator to prevent exposing those abstract intermediate classes, though.
Otherwise we will polute the View.* with non-usable controls

Reply:
will change it. didn't see that it was moved.

Reply:
That's right that we should change the generator a bit. Was mentioned in a issue, too. But that will be another PR... First of all the upgrade to 4.0